src/server/config/json/match.callback.js

```diff
@@ -1,14 +1,8 @@
 export default {
-  fail: {
-    successCode: 0,
-    errPoint: null,
-    err: null,
-  },
-  success: {
-    successCode: 1,
-  },
-  successSendMail: {
-    successCode: 1,
-    msg: 'send Email successfully.',
-  },
+  successSendMail: 'SendMail - Success to send an email.',
+  successResponse: 'ResponseMentoring - success Response.',
+  failSendMail: 'SendMail - Fail to send an email.',
+  matchAlreadyExist: 'RequestMentoring - Match already exist.',
+  cannotFoundMentor: 'RequestMentoring - Cannot found mentor.',
+  mongooseErr: 'An error occured at mongoose.',
```

Comment:
Let's sync up! in survey callback we used capitalized letters like
` ERR_GET_SURVEY: 'Survey Not Found - No Matching survey_id'`.
and please consider sorting~